packages/app/src/cli/services/context/identifiers-extensions.ts

```diff
@@ -222,15 +222,17 @@ async function createExtensions(
 }
 
 // karen.xie this name sucks too
-async function multipleConfigs(extension: ExtensionInstance): Promise<unknown[]> {
+export async function multipleConfigs(extension: ExtensionInstance): Promise<unknown[]> {
   const configContent = await extension.commonDeployConfig('')
   return Array.isArray(configContent) ? configContent : [configContent]
 }
 
 function mapExtensionsIdsNonUuidManaged(extensionsIdsNonUuidManaged: {[key: string]: string[]}) {
   const result: {[key: string]: string} = {}
   for (const key in extensionsIdsNonUuidManaged) {
-    if (extensionsIdsNonUuidManaged[key]!.length > 0) {
+    if (extensionsIdsNonUuidManaged[key]!.length > 1) {
+      result[key] = extensionsIdsNonUuidManaged[key]!.toString()
```

Comment:
kind of a hack :/ basically if there's more than 1 id we turn the entire array into a string (then we turn it back into an array later when we're updating the drafts 😬 ) 

there might be some other downstream effects that this causes